fennel/connectors/connectors.py

```diff
@@ -184,6 +198,18 @@ def source(
             "idleness parameter should not be passed when bounded is set as False"
         )
 
+    if sample is not None:
```

Comment:
Can we also verify that preproc columns shouldn't be part of sampling columns here?

Reply:
we are doing that in to_proto file, is there any reason to do it in 2 places?

Reply:
Added that check